src/Person/Person.js

```diff
@@ -0,0 +1,12 @@
+import React, { Children } from 'react';
+
+const Person = (props) => {
```

Comment:
Since you do not know the following entity specification, I would suggest to rename component to Card. Also you can pass data to card as a separate single object, containing caption and text properties as requested. This will unbind you from confusion that could appear once you use your component over unexpected data set.

Also, just a bit of syntactic sugar - if you declare a single argument expected - you can remove parenthesis.
Also you can use arrow function syntax in this way also:
`const func = arg => <div>...</div>;`